.github/workflows/ci.yml

```diff
@@ -14,6 +14,7 @@ jobs:
           - '2.7'
           - '3.0'
           - '3.1'
+          - '3.2'
```

Comment:
@petergoldstein can you please update `on` in this file to [include `pull_requests` and `workflow_dispatch`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) as well? I think that's what's keeping CI from running on your PR.

Reply:
@codealchemy Done, although now it needs approval from a maintainer to run.